webapp/channels/src/components/admin_console/bleve_settings.tsx

```diff
@@ -157,7 +175,7 @@ export default class BleveSettings extends AdminSettings {
                         <div className='job-table-setting'>
                             <JobsTable
                                 jobType={JobTypes.BLEVE_POST_INDEXING}
-                                disabled={!this.state.canPurgeAndIndex || this.props.isDisabled}
+                                disabled={!this.state.canPurgeAndIndex || Boolean(this.props.isDisabled)}
```

Comment:
why did you had to wrap into Boolean?

Reply:
I had to wrap that because `props.isDisabled` is an optional property (coming from BaseProps in admin_settings.tsx), but the `disabled` prop in JobsTable is a required boolean. Without the wrapping it throws an error due to not accepting undefined. Let me know if I should update the code here

Reply:
oh ok thank you for explaining